RecipesLyricsMashup/RecipesLyricsMashupV2/RecipeLyricsMashup/RecipeLyricsMashup/Pages/Index.cshtml

```diff
@@ -14,47 +15,51 @@
 </div>
 
 
-<form action="" method="GET">
+<form action="" method="GET" >
 
-    First name: <input type="text" name="search">
+    First name: <input type="text" name="search" value=@search> @*Morgan added "value" tag to repopulate the field with the last searched string value*@
 
     <input type="submit" value="Go" />
 
 </form>
 
-<table>
-    <tr>
-        <th>Title</th>
-        <th>Ingredients</th>
-        <th>Link</th>
-    </tr>
-    @{ 
-        foreach (QuickTypeRecipe.Result result in recipePuppyJson)
+<table class="table table-striped"> @*Morgan added*@
+    <thead class="thead-dark"> @*Morgan added*@
+        <tr>
+            <th>Title</th>
+            <th>Ingredients</th>
+            <th>Link</th>
+        </tr>
+    </thead> @*Morgan added*@
+    @{
+        foreach (QuickTypeRecipe.Result result in recipePuppyRecipies)
         {
             <tr>
                 <td>@result.Title</td>
                 <td>@result.Ingredients</td>
-                <td>@result.Href</td>
+                <td><a href="@result.Href">@result.Href</a></td> @*Morgan added*@

```

Comment:
Also, edited the URL column in each of your tables to generate clickable links based on the populated Json result.

Reply:
Ok , good.